Makefile

```diff
@@ -24,14 +30,24 @@ DATA_built  = $(RELEASE_SQL) \
 
 DOCS        = README.pg_sphere COPYRIGHT.pg_sphere
 REGRESS     = init tables points euler circle line ellipse poly path box index \
-			  contains_ops contains_ops_compat bounding_box_gist gnomo healpix \
-			  moc mocautocast epochprop
+              contains_ops contains_ops_compat bounding_box_gist gnomo
+
+ifneq ($(USE_HEALPIX),0)
+REGRESS    += healpix moc mocautocast
+endif
+
+REGRESS    += epochprop
```

Comment:
I propose not to separate epochprop into a new line. It is already included unconditionally.

Reply:
I didn't want to change the order of the tests, as I thought that might affect something. I think I saw a comment somewhere saying that "the order matters" and not to change it, but maybe that doesn't apply to `REGRESS`. If it's ok to change the order of the tests, it would certainly simplify things if I could add it unconditionally above this line and I will update the PR.

Reply:
I may guess that epochprop independent on other tests. I tried to move epochprop back and forth and I haven't seen any problems with it. I got your point. It is up to you to keep PR unchanged. Furthermore, we are planning to cleanup and improve Makefile in the future. I do not see any other problems with your PR at the moment.